LICENSE

```diff
@@ -1,5 +1,5 @@
 Mixxx version 2.3, Digital DJ'ing software.
-Copyright (C) 2001-2021 Mixxx Development Team
+Copyright (C) 2001-2022 Mixxx Development Team
```

Comment:
meeeh, #4596 should have been merged into 2.3
Can you please try to cherry-pick cb47295f20007fb7f89c5cf97ce995e27244cd7f?

Reply:
I wouldn't have remembered to update this here, so Thanks!